front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorCurves.tsx

```diff
@@ -429,14 +435,13 @@ export default function RollingStockEditorCurves({
     title: K
   ) => {
     if (!currentRsEffortCurve) return;
+    const rsEffortCurveCondKey = effortCurveCondKeys[title as keyof EffortCurveCondKeys];
```

Comment:
If you change the signature of the function like this (replace RollingStockSelectorParam by EffortCurveCondKeys), you don't need the cast anymore
```
 const removeAnotherRsParam = <K extends keyof EffortCurveCondKeys>(
    value: EffortCurveCondKeys[K],
    title: K
  ) => {}
```
and then 
```suggestion
    const rsEffortCurveCondKey = effortCurveCondKeys[title];
```

Reply:
If I change the type in this function, I have another type error popping on the next function (`confirmRsParamRemoval`) line 475 where I need to add an `as string` cast.
What do you prefer ?